changelog/unreleased/issue-21398.toml

```
@@ -0,0 +1,5 @@
+type = "f"
+message = "Fix an issue when was impossible to save search/dashboard after clearing the unit type"
```

Let's try to make the wording a bit clearer:

```suggestion
message = "Fix an issue causing saving searches/dashboards after clearing the unit type to fail with error."
```